python/lsst/meas/algorithms/ingestIndexReferenceTask.py

```diff
@@ -26,20 +26,39 @@
 import math
 
 import astropy.time
+import astropy.units as u
 import numpy as np
 
 import lsst.pex.config as pexConfig
 import lsst.pipe.base as pipeBase
 import lsst.geom
 import lsst.afw.table as afwTable
-from lsst.afw.image import fluxFromABMag, fluxErrFromABMagErr
+from lsst.daf.base import PropertyList
+from lsst.afw.image import fluxErrFromABMagErr
 from .indexerRegistry import IndexerRegistry
 from .readTextCatalogTask import ReadTextCatalogTask
 from .loadReferenceObjects import LoadReferenceObjectsTask
 
 _RAD_PER_DEG = math.pi / 180
 _RAD_PER_MILLIARCSEC = _RAD_PER_DEG/(3600*1000)
 
+# The most recent Indexed Reference Catalog on-disk format version.
+LATEST_FORMAT_VERSION = 1
+
+
+def addRefCatMetadata(catalog):
+    """Add metadata to a new (not yet populated) reference catalog.
+
+    Parameters
+    ----------
+    catalog - `~lsst.afw.table.SimpleCatalog`
+        Catalog to which metadata should be attached.  Will be modified
+        in-place.
+    """
+    md = PropertyList()
+    md.set("REFCAT_FORMAT_VERSION", LATEST_FORMAT_VERSION)
+    catalog.setMetadata(md)
```

Comment:
@TallJimbo : would this overwrite the rest of the metadata in the catalog? Should I change it to do this first `md = catalog.getMetadata()`?

Reply:
That would indeed be safer.  I recall thinking that the initial `PropertySet` was going to be `None` in the cases that mattered here, but I could easily believe that was only the case in the first place I used this function, and then I forgot about it in the next place.